index.js

```diff
@@ -88,6 +86,24 @@ async function fetchImageId(url) {
   }
 }
 
+async function fetchImageUrl(url) {
+  try {
+    const response = await axios(url);
+    const $ = cheerio.load(response.data);
```

Comment:
This could also be extracted as it repeats from the function above. Nevertheless it is ok for now =]

Reply:
@viniciuskneves I thought you had mentioned that there were articles with multiple images so that I would take care of this when scraping the image. It did not occur to me that the idea was to upload more than one image. I can take care of this issue.

Regarding the extraction, it makes sense to create a function to return the cheerio object.  After defining the first test scenarios and writing them, we will have a lot of space for refactoring since the responsibility of each function will be a lot clearer.